composeml/label_times.py

```diff
@@ -321,8 +339,8 @@ def sample(self, n=None, frac=None, random_state=None):
                 sample = label.sample(n=n, random_state=random_state)
                 sample_per_label.append(sample)
 
-            labels = pd.concat(sample_per_label, axis=0, sort=False)
-            return labels
+            sample = pd.concat(sample_per_label, axis=0, sort=False)
```

Comment:
i wonder if sample should update the settings like the other label transforms? the only question I have is what would the settings be if you ran sample twice though

Reply:
I think it may be helpful to update transforms. This way the same sample can be created if `random_state` was used. If ran twice both sample methods would be appended to the transforms in the settings used.

Reply:
okay, let's add sample to the transforms

Reply:
i think it's ready to merge after we do this. if you prefer, we could create a new issue and add it in a future PR too

Reply:
Okay. Yes, I'm thinking about opening a different issue for adding sample to the transforms.

Reply:
ok. i that case, ill aprove this PR